### PR TITLE
Random seed setter and session creator for reproducibility

### PR DESCRIPTION
**Pull request by terrytangyuan**
https://github.com/rstudio/keras/issues/42

### CODE REVIEW

**Comment by jjallaire** [comment]
Thanks! I think I am going to start with a method in the Keras package (which can also handle Theano and perhaps CNTK if there is a way to do it there). Then once we confirm that works well I'll migrate functions back to reticulate / tensorflow over time. I want to do it this way so as not to create a set of cascading dependencies for the keras dev version right before we announce the package.

**Comment by terrytangyuan** [comment]
Sounds good. Should I close this one for now?

**Comment by jjallaire** [comment]
Yes let's close this one for now.

**Comment by jjallaire** [comment]
Related PRs in Keras are here: https://github.com/rstudio/keras/pulls
